Documentation/spelling_wordlist.txt

```diff
@@ -712,6 +712,8 @@ runtime
 runtimes
 sNodes
 sService
+sServiceHost
+sServicePort
```

Comment:
This is for `k8sServicePort`.
Since there was already `sNodes` and `sService`, I added the two exception but I'm wondering if there is a better way to do this.

Reply:
Are you thinking of a wildcard, like `sService*`? It does not seem supported by [the sphinx extension](https://github.com/sphinx-contrib/spelling/blob/master/docs/source/customize.rst#input-options) we use :confused:. I don't know of a better way here.

Reply:
I was thinking more about adding the full word in the exceptions, e.g., `k8sServiceHost`, but this does not seem to work :(